src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/dao/HttpGoogleBillingDAO.scala

```diff
@@ -97,15 +97,15 @@ class HttpGoogleBillingDAO(appName: String, clientSecrets: GoogleClientSecrets,
     new Iam.Builder(httpTransport, jsonFactory, credential).setApplicationName(appName).build()
   }
 
-  def transferProjectOwnership(project: String, owner: String): Future[String] = {
+  override def transferProjectOwnership(project: String, owner: String): Future[String] = {
     /* NOTE: There is no work to be done here. It is up to the caller, inside their own FC stack to:
      * - add the project to the rawls db
```

Comment:
The relevant Rawls DB table (`RawlsBillingProjectTable`) requires the entry `cromwellAuthBucketUrl`. Where will that be coming from?

Reply:
oh, you're saying the `cromwellAuthBucketUrl` needs to be sent to rawls too? good to know, i'll expose that

Reply:
https://github.com/broadinstitute/gpalloc/issues/14